experimental/benchmarks/acmeair/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:14.04
+MAINTAINER Gareth Ellis <gareth.ellis@uk.ibm.com>
+RUN mkdir /node /node/Jmeter /node/node_undertest /node/node_baseline
```

Comment:
You could alternatively use https://docs.docker.com/engine/reference/builder/#volume for `node_undertest` and `node_baseline`, but this line should work as is :smile: